app/views/pages/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Please enter your postcode:</h2>
+<%= form_with url: '/postcode', method: :get do |form| %>
+  <%= form.label :postcode, "" %>
+  <%= form.text_field :postcode, required: true, placeholder: "Enter your postcode" %>
```

Comment:
For the placeholder, may be more useful to have an example there instead if we have a label elsewhere?